src/utils/graphql/queries/getApplication.query.ts

```diff
@@ -21,13 +21,26 @@ export default gql`
             ...TemplateStage
           }
         }
+        # Count PREVIEW actions so we know whether to show the option to Preview
+        templateActions(condition: { trigger: ON_PREVIEW }) {
```

Comment:
can you alias this please to `previewActions: templateAction`, although I think access to this might be restricted to backend only (or at least it's easier that way), maybe we can do something with visibility of fields on templateActions for non admin user, i'll make a note of it

Reply:
> can you alias this please to `previewActions: templateAction`

If I change this, when I try to refer to it in useLoadApplication, I get a type error on `hasPreviewActions: application.template.previewActions.totalCount > 0,` as its expecting auto-generated back-end "Application" type for this structure which has "templateActions" not "previewActions" on "application.template".

Any idea how to fix?

Reply:
There is a cast further up `const application = data.applicationBySerial as Application`, which kind of negates query specific types that were auto generated, if I remove the `as` it seems to work, but get some compile errors in the ActivityLog (it has EXTENSION now, I think it's just some backend/frontend mismatch)

Reply:
Okay, thanks -- think this is good now: 672001f7